packages/snaps-sdk/src/jsx/components/Box.ts

```diff
@@ -17,6 +17,25 @@ export type BoxProps = {
 
 const TYPE = 'Box';
 
+export const BOX_CHILD_TYPES = [
```

Comment:
Can we extract this from the validation struct instead? Otherwise we need some solution to keep the two in sync at least.

Reply:
We can extract it for most of the structs apart from the `Field` struct due to the tuple used in it. It makes the validation of the component impossible since we drop one element at a time.